crypto/evp/evp_enc.c

```diff
@@ -1129,11 +1129,26 @@ int EVP_CIPHER_CTX_ctrl(EVP_CIPHER_CTX *ctx, int type, int arg, void *ptr)
             return 0;
         params[0] = OSSL_PARAM_construct_size_t(OSSL_CIPHER_PARAM_IVLEN, &sz);
         break;
-    case EVP_CTRL_GCM_SET_IV_FIXED:
+    case EVP_CTRL_AEAD_SET_IV_FIXED:
         params[0] =
             OSSL_PARAM_construct_octet_string(OSSL_CIPHER_PARAM_AEAD_TLS1_IV_FIXED,
                                               ptr, sz);
         break;
+    case EVP_CTRL_GCM_IV_GEN:
+        set_params = 0;
+        if (arg < 0)
+            sz = 0; /* special case that uses the iv length */
+        params[0] =
+            OSSL_PARAM_construct_octet_string(OSSL_CIPHER_PARAM_GCM_GET_IV_GEN,
+                                              ptr, sz);
+        break;
+    case EVP_CTRL_GCM_SET_IV_INV:
+        if (arg < 0)
+            return 0;
+        params[0] =
+            OSSL_PARAM_construct_octet_string(OSSL_CIPHER_PARAM_GCM_SET_IV_INV,
+                                              ptr, sz);
+        break;
```

Comment:
I see no reason to insist on GCM control names when they are just aliases for AEAD control names...

Reply:
@levitte - EVP_CTRL_GCM_IV_GEN & EVP_CTRL_GCM_SET_IV_INV are existing names without aliases.. I am not sure what you mean.

Reply:
Oh... Darn, sorry, didn't look closely enough, I had it in my mind that it was the same with those as with iv_fixed. Never mind.